### PR TITLE
Upgrade chcker

### PR DESCRIPTION
**Pull request by ocnix**
checks if there is a new version before start up (won't work if you test since i have not created a release yet with the master hash)

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -68,7 +72,7 @@ func main() {
 	rootCmd.AddCommand(editCmd)
 	rootCmd.AddCommand(initCmd)
 	rootCmd.AddCommand(runCmd)
-	rootCmd.AddCommand(upgradeCmd)
+	//rootCmd.AddCommand(upgradeCmd)
```
Comment (miles-) [inline cmd/main.go:75]: Should we remove this comment since we won't be adding an upgrade command?

File: cmd/main.go [hunk 2]
```diff
@@ -145,6 +149,24 @@ func run(conf *config.Config, choice string) {
...
+	resp, err := http.Get(vers)
+	if err != nil{
+		fmt.Printf("Internal error on startup, could not check version: %v\n", err)
+		return
+	}
+	dat, _ := ioutil.ReadAll(resp.Body)
+	githubResp := &GithubAPIResponse{}
+	json.Unmarshal(dat, githubResp)
+	if githubResp.TagName != Version {
+		fmt.Printf("🐢 There is a new version of bifrost availabe\n🐢 run ./uninstall --upgrade to get newest version \n")
```
Comment (miles-) [inline cmd/main.go]: Should this be `./install --upgrade` instead?

File: cmd/main.go (hunk 1 above)
Comment (ocnix) [reply cmd/main.go:75]: it technically won't run, i thought i'd leave just incase, but its toggled off

File: cmd/main.go (hunk 2 above)
Comment (ocnix) [reply cmd/main.go]: yes it should be!

**Review by miles- — APPROVED** [review]
LGTM

**Pull request merged**